src/server/commands.rs

```diff
@@ -124,11 +123,37 @@ pub enum Command {
         path: String,
     },
     /// The `LIST` command
+    // This command causes a list to be sent from the server to the
+    // passive DTP.  If the pathname specifies a directory or other
+    // group of files, the server should transfer a list of files
+    // in the specified directory.  If the pathname specifies a
+    // file then the server should send current information on the
+    // file.  A null argument implies the user's current working or
+    // default directory.  The data transfer is over the data
+    // connection in type ASCII or type EBCDIC.  (The user must
```

Comment:
Don't we not support ascii type for now?

Reply:
Comes from the spec, we should check that we are compliant :-)